docs/blog/newsletter-june-2018.rst

```diff
@@ -0,0 +1,122 @@
+.. post:: June 11, 2018
+   :tags: newsletter, job-titles, error-messages, product-managers
+
+
+########################################
+Write the Docs Newsletter – June 2018
+########################################
+
+Hi folks! Kelly O. here with the last issue of the newsletter until August (in the meantime, the editorial team will be taking a wee summer break). It's been an exciting spring, with `another successful Portland conference <http://www.writethedocs.org/conf/portland/2018/news/thanks-recap/>`_ under our belts, and a jam-packed rest of the year laid out ahead of us!
+
+As a quick refresher, here's what's on the docket:
+
+**August 18-22 – Write the Docs + Open Help – Cincinnati, Ohio, USA**
+`Submit a talk <http://www.writethedocs.org/conf/cincinnati/2018/cfp/>`_ by next Monday, June 18 or `buy a ticket <http://www.writethedocs.org/conf/cincinnati/2018/tickets/>`_ any time!
+
+**September 9-11 – Write the Docs Prague – Prague, Czech Republic**
+Speakers will be announced soon, and in the meantime, you can `get your tickets here <http://www.writethedocs.org/conf/prague/2018/tickets/>`_!
+
+**November 15-16 – Write the Docs Australia – Melbourne, Australia**
+`Get your talk proposal in <http://www.writethedocs.org/conf/australia/2018/cfp/>`_ by July 20th and `snag your ticket early <http://www.writethedocs.org/conf/australia/2018/tickets/>`_, so you can start planning your visit!
+
+If you're interested in a more virtual way to engage with the community, you could also think about volunteering to help out with the year-round organization. When we come back in August, we'll be putting out a call for volunteers to help both with the newsletter team as well as a few other working groups. So have a think about whether and how you might want to pitch in.
+
+One other quick thing to note was an interesting tidbit from the `#meta <https://writethedocs.slack.com/messages/meta>`_ channel about the pros and cons of Slack's message limit. Since it's a topic that comes up on a relatively regular basis, we thought it'd be worth addressing here. Essentially, while it can be frustrating to see valuable conversations and resources swallowed by the void, the message limit helps foster a more comfortable space for community conversation, without the looming knowledge that every word you write will be preserved, publicly, for all time.
+
+We do try to offset the transience of the Slack discourse at large with this here newsletter, where we capture a handful of pithy conversations each month. Speaking of which, let's get to em!
```

Comment:
though about that, but since we're taking a little break, and we probably won't include anything that's tagged in the next few weeks, the timing seemed a little funny